server/chat-plugins/room-events.ts

```diff
@@ -18,7 +18,7 @@ function formatEvent(event: {eventName: string, date: string, desc: string, star
 	}
 	let ret = `<tr title="${explanation}">`;
 	ret += Chat.html`<td>${event.eventName}</td>`;
-	ret += Chat.html`<td>${event.aliases ? event.aliases.join(", ") : ""}</td>`;
+	ret += event.aliases ? Chat.html`<td>${event.aliases.join(", ")}</td>` : ""
```

Comment:
```suggestion
	ret += event.aliases.length ? Chat.html`<td>${event.aliases.join(", ")}</td>` : "";
```

Reply:
Also, you need to figure out a way to include empty `<td>` tags when displaying all events if one event has alias(es) and others do not - I'll leave that for you to figure out (maybe pass an argument to `formatEvent()`?). Currently, your code doesn't work when showing all aliases.